### ISSUE TITLE
Site Owners: Sites List

### ISSUE THREAD

**Issue by jlguillen** [SUNET/sri-front#233]
Site owners - when choosing site owner does not show list of sites connected to that owner. Should be 'filterable'.

**Comment by PaKZer0** [comment]
See #274 we need to solve this issue before marking this as done.

**Comment by mariahaider** [comment]
I can see the sites now.

**Comment by PaKZer0** [comment]
Yes but the filter only filters by name, we need to add the description too.

**Comment by jlguillen** [comment]
no description in the Site form

**Comment by PaKZer0** [comment]
We'll need to filter over country and owner id values.
